client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -10,7 +10,6 @@ import { useEffect, useState, useCallback } from 'react';
 import Modal from 'react-modal';
 import { Switch, Route, Redirect, generatePath, useHistory, useLocation } from 'react-router-dom';
 import DocumentHead from 'calypso/components/data/document-head';
-import WordPressLogo from 'calypso/components/wordpress-logo';
```

Comment:
This was the reason for the failure. p1678083382871709-slack-CRWCHQGUB
I missed it 😣

Reply:
No worries. It's my fault 😓